Includes/outputLine.cpp

```diff
@@ -1,5 +1,13 @@
 #include "outputLine.h"
 
+#include <cstdarg>
+#include <cstdio>
+#include <SDL.h>
```

Comment:
No need for SDL.h if building with NXDK, or am I missing something?

Reply:
Right, changed.